src/main/scala/com/moesif/filter/MoesifApiFilter.scala

```diff
@@ -274,7 +275,13 @@ class MoesifApiFilter @Inject()(config: MoesifApiFilterConfig)(implicit mat: Mat
           if (ex.getMessage.contains("failed to respond") || ex.getMessage.contains("api-dev.moesif.net:443")) { // for unirest publishResponse error
             printEventBatchRequestHelper(context)
           }
-//        TODO check time out exception
+
+          ex match {
```

Comment:
Can we reduce log by adding var isTimedout : Boolean = False

and add it to the same logger timeout = ${isTimeout}.

nvm.. leave it as it's temp change.